contrib/integrations/kubernetes/plugin-kubernetes/main.go

```diff
@@ -277,38 +278,47 @@ func executeK8s(q *integrationplugin.DeployQuery) error {
 }
 
 func executeHelm(q *integrationplugin.DeployQuery) error {
-	releaseName := q.GetOptions()["cds.integration.release_name"]
-	namespace := q.GetOptions()["cds.integration.namespace"]
-	helmChart := q.GetOptions()["cds.integration.helm_chart"]
-	helmValues := q.GetOptions()["cds.integration.helm_values"]
-	timeoutStr := q.GetOptions()["cds.integration.timeout"]
 	project := q.GetOptions()["cds.project"]
 	workflow := q.GetOptions()["cds.workflow"]
-	application := q.GetOptions()["cds.application"]
-	if namespace == "" {
-		namespace = "default"
+	helmVersion := q.GetOptions()["cds.integration.helm_version"]
```

Comment:
Can you add a check on the value of helm_version, as:

```go
v, err := semver.Parse(helmVersion)
		if err != nil {
			return fmt.Errorf("Invalid Helm Version %s - err: %v, helmVersion, err)
		}
```

semver import is `"github.com/blang/semver"`